ndb_api/profile_helpers.go

```diff
@@ -86,15 +87,26 @@ func ResolveProfiles(ctx context.Context, ndb_client *ndb_client.NDBClient, data
 	// DB Param Profile
 	dbParam, err := dbParamProfileResolver.Resolve(ctx, dbEngineSpecific, DbParamOOBProfileResolver)
 	if err != nil {
-		log.Error(err, "DbParam Profile could not be resolved", "Input Profile", networkProfileResolver)
+		log.Error(err, "DbParam Profile could not be resolved", "Input Profile", dbParamProfileResolver)
 		return nil, err
 	}
 
+	// DB Param Instance Profile
+	dbParamInstance, err := dbParamInstanceProfileResolver.Resolve(ctx, dbEngineSpecific, DbParamOOBProfileResolver)
+	if err != nil {
+		// Database Parameter Instance profile is required only for sql server
+		if databaseType == common.DATABASE_TYPE_SQLSERVER {
```

Comment:
possible duplication: MSSQL/SQLSERVER, please check.

Reply:
Fixed, thanks for catching this